public/app/plugins/datasource/mysql/partials/config.html

```diff
@@ -27,6 +27,23 @@ <h3 class="page-heading">MySQL Connection</h3>
 		</div>
 	</div>
 
+  <div class="gf-form max-width-30">
+    <span class="gf-form-label width-10">Session Timezone</span>
+    <input
+            type="text"
+            class="gf-form-input gf-form-input--has-help-icon"
+            ng-model="ctrl.current.jsonData.timezone"
+            spellcheck='false'
+            placeholder="(default)"
+    ></input>
+    <info-popover mode="right-absolute">
+      Specify the time zone used in the database session, e.g. <code>Europe/Berlin</code> or <code>+02:00</code>.
+      The value will be set int the session with <code>SET time_zone='...'</code>. If you leave this field empty,
```

Comment:
Might want to add here and/or in the docs why and in what scenarios configuring this is a good idea.

```suggestion
      The value will be set in the session with <code>SET time_zone='...'</code>. If you leave this field empty,
```

Reply:
I added some information on why the configuration is necessary. Can you please have another look? Thanks!